tests/BenchmarkDotNet.IntegrationTests/RoslynToolchainTest.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Globalization;
+using System.Threading;
+
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Tests.Loggers;
+using BenchmarkDotNet.Toolchains.Roslyn;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace BenchmarkDotNet.IntegrationTests
+{
+    public class RoslynToolchainTest : BenchmarkTestExecutor
+    {
+        public RoslynToolchainTest(ITestOutputHelper output) : base(output) { }
+
+        /// <summary>Prooftest for #1039.</summary>
+        [TheoryFullFrameworkOnly]
```

Comment:
I am sorry, my previous suggestion was incomplete: we need to specify namespace (or add a using to the file) and the reason why we want to skip

```suggestion
        [Tests.XUnit.TheoryFullFrameworkOnly("Roslyn toolchain does not support .NET Core")]
```